webcompat/templates/home-page/form.html

```diff
@@ -62,7 +62,8 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
             </span>
           </div>
           <div class="wc-Form-group js-Form-group">
-            {{ form.steps_reproduce.label(class_='wc-Form-label') }} {{ form.steps_reproduce(class_='wc-Form-item wc-Form-item--textarea js-autoExpand', rows=4) }}
+            <label class="wc-Form-label" for="steps_reproduce">How did you get there?</label>
```

Comment:
We don't need to hardcode this. `form.steps_reproduce(class_='wc-Form-item wc-Form-item--textarea js-autoExpand', rows=4)` <-- you should be able to pass in `placeholder=foo`.